docs/x509/tutorial.rst

```diff
@@ -57,14 +57,15 @@ a few details:
 .. code-block:: pycon
 
     >>> from cryptography import x509
+    >>> from cryptography.x509 import oid
```

Comment:
This is inconsistant with how we did this import everywhere else. Why not just import `NameOID` directly?